docs/release/release_0_4_17.md

```diff
@@ -235,19 +273,19 @@ https://github.com/napari/napari
 - Fix broken links in Napari Code of Conduct (#5005)
 - Fix sys.path issue with subprocess relaunch in macOS (#5007)
 - MAINT: update issue failing template (#5012)
+- Add link to docs artifact to pull requests (#5014)
 - [pre-commit.ci] pre-commit autoupdate (#5015)
 - Feature: register a keyboard shortcut for `preserve_labels` checkbox (#5017)
+- Add option to edit second shortcut in settings (#5018)
+- Fix shift selection for points (#5022)
 - Revert "Fix sys.path issue with subprocess relaunch in macOS" (#5027)
+- Use Source object to track if a layer is duplicated (#5028)
 - Revert "Revert "Fix sys.path issue with subprocess relaunch in macOS"" (#5029)
-- Option to load all dock widgets (#4954)
-- Ensure non-negative edge width (#5035)
+- Add release notes for 0.4.17 (#5031)
 - MAINT: fix a couple of trans._. (#5034)
-- Fix shift selection for points (#5022)
-- Add saving folders to preferences (#4902)
-- Use Source object to track if a layer is duplicated (#5028)
-- Add saving folders to preferences (#4902)
-- Add option to edit second shortcut in settings (#5018)
-- Update viewer screenshot in docs and make more prominent on napari.org (#4988)
+- Ensure non-negative edge width (#5035)
+- add #4954 to release notes (#5038)
```

Comment:
Not sure if we should include release note updates in the release notes, but I'm lazy and just kept them for now.